File: programasjava/02-EstructurasDeControl/VerificacionAprobado.java
import javax.swing.JOptionPane;
public class VerificacionAprobado {
    public static void main(String[] args) {
        
        // Declaración de variables
        double promedio=0.0, calif1=0.0,
                calif2=0.0, calif3=0.0;

        // Solicitar las calificaciones y calcular el promedio
        calif1 = Double.parseDouble
                (JOptionPane.showInputDialog("Introduce la calif 1"));
        calif2 = Double.parseDouble
                (JOptionPane.showInputDialog("Introduce la calif 2"));
        calif3 = Double.parseDouble
                (JOptionPane.showInputDialog("Introduce la calif 3"));

        promedio = (calif1 + calif2 + calif3)/3;

        if(promedio >= 7.0){
            JOptionPane.showMessageDialog(null,
                    "Aprobado");
        }
    }
}

File: programaspython/02-estructurasdecontrol/verificacion_aprobado.py

# Solicitud de calificaciones

calif1 = float(input("Introduce la calificación 1\n"))
calif2 = float(input("Introduce la calificación 2\n"))
calif3 = float(input("Introduce la calificación 3\n"))

promedio = (calif1 + calif2 + calif3) / 3

if promedio >= 7.0:
    if promedio>=7.1 and promedio<=7.5:
        print('Aprobado Bajo')
    elif promedio>=7.6 and promedio<=8.7:
        print('Aprobado Media')
    else:
        print('Aprobado Alto')
else:
    print('No Aprobado')

